README.md

```diff
@@ -1,22 +1,50 @@
-# Azure Kubernetes Service (AKS) Baseline Cluster
+# Azure Kubernetes Service (AKS) Multi Cluster
 
-This reference implementation demonstrates the _recommended starting (baseline) infrastructure architecture_ for a general purpose [AKS cluster](https://azure.microsoft.com/services/kubernetes-service). This implementation and document is meant to guide an interdisciplinary team or multiple distinct teams like networking, security and development through the process of getting this secure baseline infrastructure deployed and understanding the components of it.
+This reference implementation will go over some design decisions from the baseline to detail them as a well as incorporate some new _recommended infrastructure options_ for a Multi Cluster architecture. In this oportunity, this implementation and document are meant to guide the multiple distinct teams as introduced in the [AKS Baseline](https://github.com/mspnp/aks-secure-baseline) through the process of expanding from a single cluster to multiple clusters solution with a foundamental driver in mind which is **High Availability**.
```

Comment:
```suggestion
This reference implementation will go over some design decisions from the baseline to detail them as a well as incorporate some new _recommended infrastructure options_ for a multi-cluster AKS architecture. This implementation and document are meant to guide the multiple distinct teams as introduced in the [AKS Baseline](https://github.com/mspnp/aks-secure-baseline) through the process of expanding from a single cluster to a multi-cluster solution with a fundamental objective of **High Availability**.
```

Reply:
I agree 100% that some content in here is too much for an overview, and we should move that to other sections. We plan to keep moving things around in the following days. 

Thanks for the great co-authoring edition that improved how it reads 10X 😹